cogs/boolean_utils.py

```diff
@@ -0,0 +1,395 @@
+"""
+discord bot cog that is used for various number utility functions
+"""
+import discord
+from discord.ext import commands
+
+class BoolUtilsCog:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def bool(self, ctx, exp):
```

Comment:
Actually, if you change this to `async def bool(self, ctx, *, exp):` you don't need to wrap in quotes. It's in the discord API docs, but also can be seen here: https://github.com/Chris-Johnston/CSSBot_Py/blob/master/cogs/courseInfo.py#L59

Reply:
Huh, that is pretty neat. I'll change that